compiler-rt/test/asan/TestCases/inline.cpp

```diff
@@ -2,10 +2,14 @@
 
 // Test that no_sanitize_address attribute applies even when the function would
 // be normally inlined.
+//
+// XFAIL: MSVC
+// ^ MSVC has this bug
```

Comment:
which bug?

Reply:
if the function is inlined the no_sanitize_ address equivalent in msvc (`__declspec(no_sanitize_address)`) does not apply